apps/condo/domains/user/access/CheckUserExistenceService.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Generated by `createservice user.CheckUserExistenceService --type queries`
+ */
+
+async function canCheckUserExistence () {
+    return true
```

Comment:
(optional, just strange idea) Seems strange, but we can limit usage to only unauthorised users. Like reverse default logic

Reply:
What is this for?
I have never come across such logic anywhere